.pipelines/ci.yml

```diff
@@ -21,22 +21,22 @@ jobs:
 
     strategy:
       matrix:
-        py310-1131:
+        py310-1141:
           python.version: '3.10'
           torch.version: 'torch torchvision torchaudio'
-          ort.version: '1.13.1'
-        py39-1121:
+          ort.version: '1.14.1'
+        py39-1131:
           python.version: '3.9'
           torch.version: 'torch==1.12.1+cpu torchvision==0.13.1+cpu torchaudio==0.12.1+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html'
-          ort.version: '1.12.0'
-        py38-1110:
+          ort.version: '1.13.1'
+        py38-1121:
           python.version: '3.8'
           torch.version: 'torch==1.11.0+cpu torchvision==0.12.0+cpu torchaudio==0.11.0+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html'
-          ort.version: '1.11.0'
-        py37-1100:
+          ort.version: '1.12.1'
+        py37-1110:
```

Comment:
should this be `py37-1111` (for version 1.11.1)? if so, it also needs to be changed everywhere else `py37-1110` is used.

Reply:
Good finding. But given the pipeline was successfully running for a while, let's keep it as it was, and it will be dropped in the next version.

Reply:
yes, sounds good!